packages/oak-addon-basic-components/lib/core/Editor/ColorButton.js

```diff
@@ -75,9 +75,23 @@ export default ({ className }) => {
       <DropdownMenu>
         <ColorField
           ref={colorFieldRef}
-          className="oak-color-field py-3"
+          className="py-3"
           value={color}
           onChange={onChange}
+          popperOptions={{
+            strategy: 'relative',
+            modifiers: [{
```

Comment:
is this modifier really necessary?

Reply:
Otherwise the popper will be 'absolute' by default and won't be inside de dropdown.

Reply:
using `strategy: 'relative'` should be enough

Reply:
I thought you were talking about the strategy prop 😅
yes the offset is enough to deal with the transform